solidity/test/token_stake/TestTokenStake.js

```diff
@@ -305,6 +305,16 @@ contract('TokenStaking', function(accounts) {
       )
     })
 
+    it("should not permit undelegating at the time when initialization " +
```

Comment:
We should permit undelegating when initialization period passed and we should not permit before. And we have test cases for those scenarios declared just before this one:
- should permit undelegating at the time when initialization period passed
- should not permit undelegating at the time before initialization period passed

Reply:
ah sorry I got it wrong then, do you remember what corner case did you mean in the previous review?

Reply:
That comment was all about removing `-1 / +1` from `increaseTime / increaseTimeTo` so that both mentioned tests use margin values. I looked at the code and it's all correct now so I think we can just drop the test from line 308.